### PR TITLE
fix: undefined search tag, sidebar colors #9789, #9811

### LINKED ISSUE #9789: Frio: undefined Variable when searching for non existent hashtag
### Bug Description

Searching for a hashtag which doesn't exist on the instance outputs `undefined` in Frio as result.
I guess it's because of an undefined js variable.

Expected Result:

If there are no search results the output should be rather something like "no results" than "undefined"

### Platform Info

Friendica Version:
2021.01

### LINKED ISSUE #9811: Theme: frio: mobile: low prio: space below right sidebar does not apply background color
In frio dark theme on mobile the space below the right sidebar does not apply background color and is displayed in white. This looks especially odd for dark and black theme.

Expected: should have same background color used in rest of theme.

white space

### PR DESCRIPTION
**Pull request by Extarys**
Fixes #9789 #9811

### CODE REVIEW

**Comment by Extarys** [comment]
Hmm github didn't catch that I added #9811 too. I should've made 2 PR on 2 different branches...

File: view/theme/frio/css/style.css [hunk 1]
```diff
@@ -2045,14 +2073,19 @@ ul.tabbar ul.tabs-extended li.active a {
...
+    width: 100%;
+    box-sizing: border-box;
 }
-
-
 /* Dropdown Menu */
 .dropdown-menu li a,
 .dropdown-menu li .btn-link {
-	color: $link_color;
+    /* color: $link_color; */
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: Please remove commented out properties. The git history will allow us to find previous code.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/css/style.css (hunk 1 above)
Comment (Extarys) [reply view/theme/frio/css/style.css]: Yeah that was for testing, again. I hope I won't always leave comments behind :joy:

**Review by MrPetovan — APPROVED** [review]

File: view/theme/frio/css/style.css (hunk 1 above)
Comment (MrPetovan) [reply view/theme/frio/css/style.css]: Don’t worry, I’ll point them out every single time! 😁

**Pull request merged**